README.md

```diff
@@ -342,11 +380,13 @@ Lets assume you want to open the details of b.
 As the menu is below the main menu, the resulting ActionCode here is 'details-b'.
 
 With manual you would use a root action:
+
 ```js
-menu.manual('Open details of b', 'details-b', {root: true})
+menu.manual('Open details of b', 'details-b', { root: true })
```

Comment:
JS Standard Code style has spaces within brackets, XO Code Style has not… I like the spaces but I try to be consistent with XO.

```suggestion
menu.manual('Open details of b', 'details-b', {root: true})
```